dapp/src/sections/AuctionCarousel.tsx

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2025 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Button, ButtonSize, ButtonType } from '@iota/apps-ui-kit';
+import { useRouter } from 'next/navigation';
+import { useCallback, useState } from 'react';
+import Carousel from 'react-multi-carousel';
+
+import 'react-multi-carousel/lib/styles.css';
+
+import { AuctionBidDialog, AuctionDetails } from '@/auctions';
+import { AuctionPublicItem } from '@/auctions/components/AuctionPublicItem';
+import { useAvailabilityCheckDialog } from '@/stores/useAvailabilityCheckDialog';
+
+interface AuctionCarouselProps {
+    auctions: AuctionDetails[];
+    isLoading?: boolean;
+}
+
+// Responsive breakpoints for the carousel
+const responsive = {
+    desktop: {
+        breakpoint: { max: 3000, min: 1600 },
+        items: 5,
+        slidesToSlide: 1,
+    },
+    tabletLg: {
+        breakpoint: { max: 1600, min: 1024 },
+        items: 4,
+        slidesToSlide: 1,
+    },
+    tablet: {
+        breakpoint: { max: 1024, min: 768 },
+        items: 3,
+    },
+    mobile: {
+        breakpoint: { max: 768, min: 0 },
+        items: 1,
+    },
+};
+
+export function AuctionCarousel({ auctions, isLoading }: AuctionCarouselProps) {
+    const [bidDialogName, setBidDialogName] = useState<string | null>(null);
+
+    const onBidClick = useCallback((name: string) => {
+        setBidDialogName(name);
+    }, []);
+
+    if (isLoading) {
+        return (
+            <div className="w-full">
+                <AuctionCarouselHeader />
+                <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-4 gap-4">
+                    {Array.from({ length: 4 }).map((_, index) => (
+                        <div key={index} className="animate-pulse">
+                            <div className="bg-names-neutral-12 rounded-lg h-64"></div>
+                        </div>
+                    ))}
+                </div>
+            </div>
+        );
+    }
+
+    if (!auctions.length) {
+        return (
+            <div className="w-full">
+                <AuctionCarouselHeader />
+                <div className="text-center py-12">
+                    <p className="text-body-lg text-names-neutral-70">
+                        No active auctions at the moment
+                    </p>
+                </div>
+            </div>
+        );
+    }
+
+    return (
+        <div className="w-full">
+            <AuctionCarouselHeader />
+
+            <style jsx global>{`
+                .react-multi-carousel-track {
+                    gap: 0;
```

Comment:
```suggestion
                    gap: 24px;
                    padding: 8px;
```
We could add a gap and some padding so the box-shadow is more visible and there’s a separation similar to the one in Figma, like this: 
<img width="1585" height="771" alt="image" src="https://github.com/user-attachments/assets/ce12f676-cfac-46d0-9606-6e1c9e21e1bf" />